pkg/scheduler/actions/backfill/backfill.go

```diff
@@ -72,13 +74,25 @@ func (backfill *Action) Execute(ssn *framework.Session) {
 				for _, node := range ssn.Nodes {
 					// TODO (k82cn): predicates did not consider pod number for now, there'll
 					// be ping-pong case here.
-					if err := ssn.PredicateFn(task, node); err != nil {
-						klog.V(3).Infof("Predicates failed for task <%s/%s> on node <%s>: %v",
+					// Only nodes whose status is success after predicate filtering can be scheduled.
+					var statusSets util.StatusSets
+					statusSets, err := ssn.PredicateFn(task, node)
+					if err != nil {
+						klog.V(3).Infof("backfill predicates failed for task <%s/%s> on node <%s>: %v",
```

Comment:
please make the log clear for user. for example. change the `backfill predicates failed xxx` to `predicate failed in backfill action xxx`

Reply:
done